pkg/httpRequest/request.go

```diff
@@ -0,0 +1,20 @@
+package httpRequest
+
+import (
+	"net/http"
+	"time"
+)
+
+var HTTPClient = &http.Client{
+	Timeout: 5 * time.Second,
+}
+
+func HTTPDo(req *http.Request) (*http.Response, error) {
```

Comment:
So the function name should be relative with package name. The package name is `httpclient` so this function should be Get() or Post() like that becase when another package called like `httpclient.Get()` it should be understand than `httpclient.HTTPDo()`.

Reply:
อันนี้ทำหน้าที่แค่ส่ง request ที่รับเข้ามา ส่วนตัวที่กำหนด GET/POST คือ new request ที่รับเข้ามาครับ
<img width="664" alt="Screenshot 2567-09-01 at 15 40 58" src="https://github.com/user-attachments/assets/0c74c8ce-2574-4919-9bad-73b8ef590edc">
จาก code ตัว client ทำหน้าที่ post แล้ว HTTPDo ก็เป็นตัวส่ง